apps/docs/src/app/docs/indexer/page.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Indexer
+nextjs:
+  metadata:
+    title: Indexer
+    description: Blobscan indexer
+---
+
+Here you can find information related to the indexer used by the Blobscan explorer.
+
+See the repository [here](https://github.com/Blobscan/blobscan-indexer.rs/tree/master).
+
+## How it works?
+
+The indexer traverses the Ethereum blockchain, retrieving data from both the Execution and Consensus layer clients. It examines each slot's execution payload in the Consensus Layer to identify blocks with blob versioned hashes. Afterward, it retrieves the blob data from the Consensus Layer, the remaining block and tx data from the Execution Layer client, and forwards it to the Blobscan API. The data is then stored in a PostgreSQL database, while the blobs are persisted in alternative storage solutions.
+
+## How to run locally?
+
+1. Install dependencies
+
+```bash
+sudo apt install libssl-dev
+```
+
+2. Git clone this repository.
+
+```bash
+git clone https://github.com/Blobscan/blobscan-indexer.rs.git
+cd blobscan-indexer.rs
+```
+
+3. Set the [environment variables](/docs/environment#indexer).
```

Comment:
I wrote the env vars section in a way to avoid duplicating environment variables in multiple tables. The indexes needs more env vars than those actually listed in the environment page.

Maybe we can create a markdown environmend.md file in the indexer repo, enumerating all required variables and link from here. Or linking to the .env.example file which is auto-descriptive. What do you think?

Reply:
Let's go with the .env.example link